### DISCUSSION TITLE
Forms are not showing on Draft in Craft CMS

### DISCUSSION THREAD

**Discussion by elflores-beyondtrust** [solspace/craft-freeform#956] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
When adding a form and creating a draft for the entry. The form does not show. 

Entry is returning NULL value on statement below: 

`{% set entry = elemId|default ? craft.entries.id(elemId|intVal).one() : "" %} `

**Steps to reproduce**

1. Create an entry and add a form 
2. Create a draft on the entry
3. The form does not load

**Expected behavior**
The form loads on drafts 

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Pro 3.9.4 
- Freeform Version: Pro
- Freeform Edition: 3.13.32
- Fresh Install or Upgrade: 

**Additional context**

**Comment by kjmartens** [comment, marked as answer]
Hi @elflores-beyondtrust,

I just tested this on my end, and it seems to be working fine. I tested in both Freeform 3.13.33 and 4.1.13. My code is simple:

``` twig {8}
<h1>{{ entry.title }}</h1>

{% if entry.freeformForm is defined and entry.freeformForm is not empty %}

    {% if entry.freeformForm.submittedSuccessfully %}
        <h3>Submission Received!</h3>
    {% else %}
        {{ entry.freeformForm.render() }}
    {% endif %}

{% endif %}
```
